pages/dkp/konvoy/1.8/release-notes/index.md

```diff
@@ -57,6 +65,7 @@ For more information on addressing this limit, see [Docker hub rate limits](../o
 - The auto-provisioner can be now uninstalled if the `autoProvisioning.disabled` is set to `true` after the initial installation.
 - Use the correct GPG key when installing FIPS RPM packages in an air-gapped environment.
 - Include the missing `docker.io/mesosphere/pause:3.2` needed for air-gapped FIPS installation.
+- Allow for using custom CA when generating certificates for the Kubernetes control plane by copying `ca.crt` and `ca.key` files from `extras/pki` in the working directory.
```

Comment:
This really needs to be broken into a feature section, and then linked to the new topic.